lnwallet/chainfee/minrelayfeemanager.go

```diff
@@ -0,0 +1,72 @@
+package chainfee
+
+import (
+	"sync"
+	"time"
+)
+
+// minRelayFeeManager is used to store and update the minimum relay fee that is
+// required by a transaction to be accepted to the mempool.
+// The minRelayFeeManager ensures that the backend used to fetch the fee is not
+// queried too regularly.
+type minRelayFeeManager struct {
```

Comment:
Would `minFeeManager` be a more appropriate name since we now use `minmempoolfee` for bitcoind but `minrelayfee` for btcd?